packages/effect/src/Number.ts

```diff
@@ -493,3 +493,64 @@ export const parse = (s: string): Option<number> => {
     ? option.none
     : option.some(n)
 }
+
+/**
+ * Takes a string and returns an `Option` of `number`.
+ *
+ * If the string is empty or invalid characters,
+ * it returns `option.none`. Otherwise, it attempts to convert the string to a number
+ * and returns `option.some(number)`.
+ *
+ * @param s - The string to be converted to a `number`.
+ * @returns An `Option` type that is either `some(number)` if the string can be converted
+ *          to a valid number, or `none` if the string is empty or contains invalid characters
+ *          for number conversion.
```

Comment:
```suggestion
 * @param s - The string to be converted to a `number`.
```

Redundant (repetition of function description)